### DISCUSSION TITLE
Pop Laucher [Super + /] finding nothing

### DISCUSSION THREAD

**Discussion by doorz1009** [pop-os/shell#1555] [Q&A] [answered]
I just upgraded to Ubuntu 22.04 and my Pop Launcher does not populate with any entries as I type. For example, typing out "Firefox" or "Terminal" results in no change. I have uninstalled/reinstalled the shell on the master_jammy branch and have not had any more success.

Is there some sort of config that needs to be updated? Thank you in advance!

**Comment by leviport** [comment, marked as answer]
Did you also install the Launcher? https://github.com/pop-os/launcher/

It was originally built into pop-shell, but now it's sort of a separate component that must also be installed.

**Reply by doorz1009** [reply]
I did not realize that! I'll try to get that installed and if that resolves this then I'll get it marked as answered

**Reply by doorz1009** [reply]
@leviport it works perfectly now. Thank you so much!
